File: NumbersGenerator/numb.js
function numb(){
    let start = new Date();
    let count = 0;
    while(count<=100000){
        console.log(count);
        count++;
    } 
    let end = new Date();
    const time = (end.getTime()-start.getTime())/1000;
    console.log("Current time: ", time + " seconds.")
}

numb();

File: NumbersGenerator/numb.cs
using System;

public class numb{

    public static void Main(){
        DateTime start = DateTime.Now;
        int count = 0;
        while(count<=100000){
            Console.WriteLine(count);
            count++;
        }
        DateTime end = DateTime.Now;
        TimeSpan time = end - start;
        Console.WriteLine("Current time: " + time + " secods.");
    }
    
}
